templates/scheduler-policy-config-with-extender.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
missed this in my previous review -- where are we using this file? if it's being used in the base repo (coreos-ansible) then it's better to keep it in there.